presto-hive/src/test/java/com/facebook/presto/hive/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -109,4 +112,28 @@ private static BinaryStatistics stringColumnStats(String minimum, String maximum
         statistics.setMinMax(Binary.fromString(minimum), Binary.fromString(maximum));
         return statistics;
     }
+
+    @Test
+    public void testFloat()
```

Comment:
I agree, but at least it documents the weird expectations we have for float. all of the tests in this class are of a similar ilk.